test/Dialect/FIRRTL/lower-types.mlir

```diff
@@ -1158,6 +1158,19 @@ firrtl.module private @is1436_FOO() {
     firrtl.strictconnect %bov, %x_read : !firrtl.bundle<a: vector<uint<1>,2>, b: uint<2>>
     // CHECK-NEXT: }
   }
+  // Check how rwprobe's of aggregates in instances are handled.
+  // Temporary until no longer need to lower these.
+  // CHECK-LABEL: firrtl.module private @InstWithRWProbeOfAgg
+  firrtl.module private @InstWithRWProbeOfAgg() {
+    // CHECK-NOT: firrtl.probe
```

Comment:
This confused me, doesn't it contradict the next check line? If I'm missing something, please disregard.

Reply:
`CHECK-NOT` checks the string/match doesn't occur between two positive checks (or begin/end of file), so this is just saying "no firrtl.probe's until the one in the next check (which is last due to closing paren)".

Reply:
I see, I missed the closing paren, and this behavior of CHECK-NOT. Nice!